curriculum/end2end/app.js

```diff
@@ -1 +1,19 @@
 // All your code should go here
+const fs = require('fs');
+
+const solution = () => {
```

Comment:
name your files `1.js`, `2.js`, `3.js`, etc.

Reply:
otherwise you get this nasty merge conflict

Reply:
I see. Thanks.